### PR TITLE
made a 404 page

### PR DESCRIPTION
**Pull request by killertofus**

### CODE REVIEW

File: 404.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+<html>
+
+  <head>
+    <meta charSet="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <link rel="icon" href="/favicon.ico" />
+    <title>404: This page could not be found</title>
+    <meta name="next-head-count" content="4" />
+    <link data-next-font="" rel="preconnect" href="/" crossorigin="anonymous" />
+    <link rel="preload" href="/_next/static/css/79ba91c6d57651c5.css" as="style" />
```
Comment (electron271) [inline 404.html:11]: wha

Comment (electron271) [reply 404.html:11]: we dont use nextjs

File: assets/404.css [hunk 2]
```diff
@@ -0,0 +1,1668 @@
+/*
```
Comment (electron271) [inline assets/404.css:1]: why does the 404 of all things need 1600 lines of css

**Review by electron271 — CHANGES_REQUESTED** [review]

**Comment by killertofus** [comment]
now it has less code :)

**Review by HackedOS — APPROVED** [review]
lgtm

**Comment by HackedOS** [comment]
its not letting me merge?????

**Review by electron271 — APPROVED** [review]
you cant self-review

**Comment by electron271** [comment]
no wait nevermind i thought it was your pr

was cause i requested changes

**Pull request merged**